best_practices/stac_best_practices.md

```diff
@@ -17,36 +38,95 @@ If loading from or storing to a datacube format (e.g. netCDF, ZARR, GRIB), the f
   
     Each variable should be a separate datacube, no attempt should be made to combine variables automatically.
 
-## Raster data
+## Raster Data
 
 For raster data that is not stored in a datacube format (e.g. GeoTiff, JPEG2000) the following is recommended to transform the imagery into datacubes and vice-versa.
 
 ### Loading
 
-> [!NOTE]  
-> This section is incomplete and just a list of ideas.
+When loading raster data into datacubes, the following considerations and recommendations should be implemented.
 
-For satellite imagery (e.g. Sentinel-2 L2A), it may happen that ...
+Generally, if different choices of strategies are possible, the implemented strategy should be documented in the metadata.
+The generation of the datacubes should be predictable.
 
-- ... sparse data
-- ... overlap
-- resolution: if different resolutions are available, which resolution to pick? upsample? downsample?
-- creating horizontal spatial dimensions
-- creating vertical spatial dimension
-- creating time dimensions (intended step/interval?)
-- creating bands dimension (homogeneous data types?)
-- creating other dimensions (homogeneous data types?)
+Ensure homogeneous data types for the values in the datacube, choosing the most precise data type of the data types present (e.g. float over int).
```

Comment:
I'm not sure if I'm the target audience for this document, but the text in the parenthesis gives me the impression that floats are at least as precise as ints which isn't true when dealing with large int values (https://stackoverflow.com/a/12447566).

Reply:
I guess we can remove that part.

```suggestion
Ensure homogeneous data types for the values in the datacube, choosing the most precise data type of the data types present.
```